tests/cdb2sql_options.test/runit

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+bash -n "$0" | exit 1
+
+dbname=$1
+if [[ -z $dbname ]] ; then
+    echo dbname missing
+    exit 1
+fi
+
+for testreq in `find . -name 't[0-9][0-9].sql' -exec basename {} \;` ; do
+    testname=`echo $testreq | cut -d "." -f 1`
+    cmd=$'cdb2sql -d $";\n" -s -f $testreq ${CDB2_OPTIONS} $dbname default'
+    echo "$cmd" "> $testname.output"
+    eval "$cmd" 2>&1 | perl -pe "s/.n_writeops_done=([0-9]+)/rows inserted='\1'/; s/BLOCK2_SEQV2\(824\)/BLOCK_SEQ(800)/; s/OP #2 BLOCK_SEQ/OP #3 BLOCK_SEQ/;" > $testname.output
+
+    cmd="diff $testname.expected $testname.output"
+    $cmd > /dev/null
+
+    if [[  $? -eq 0 ]]; then
+        echo "passed $testname"
+    else
+        echo "failed $testname"
+        echo "see diffs here: $HOSTNAME"
+        echo "> diff -u ${PWD}/{$testname.expected,$testname.output}"
+        echo
+        exit 1
+    fi
+done
+
+echo
+exit 0
```

Comment:
Use [compare_results.sh](https://github.com/bloomberg/comdb2/blob/master/tests/tools/compare_results.sh) instead?

Reply:
will need to change the script to start accepting additional options, the script already uses -d for database name.

Reply:
You can put it in $CDB2_OPTIONS.